build/raise-build-plugin-version/Jenkinsfile

```diff
@@ -0,0 +1,42 @@
+pipeline {
+  agent {
+    dockerfile {
+      dir 'build'
+    }
+  }
+
+  options {
+    buildDiscarder(logRotator(numToKeepStr: '10'))
+    disableConcurrentBuilds()
+  }
+
+  triggers {
+    cron '@midnight'
+  }
+  
+  parameters {
+    booleanParam name: 'dryRun', defaultValue: true, description: 'Whether the build should push the changes or not'
+    string(name: 'branch', defaultValue: 'master', description: 'Source branch e.g. master, release/8.0')
+    string(name: 'newReleaseVersion', defaultValue: '10.0.0', description: 'New project-build-plugin release version e.g 9.3.0')
+    string(name: 'newSnapshotVersion', defaultValue: '10.0.0-SNAPSHOT', description: 'New project-build-plugin SNAPSHOT version e.g 9.3.1-SNAPSHOT')
+  }
+
+  stages {
+    stage('build') {
+      steps {
+        script {
+          sh "git config --global user.name 'ivy-team'"
```

Comment:
which user and email?

Reply:
from parameters or also the ivy-team user is ok for me...